net/hev-socks5-tunnel/Makefile

```diff
@@ -0,0 +1,46 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=hev-socks5-tunnel
+PKG_VERSION:=2.7.3
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/heiher/hev-socks5-tunnel/releases/download/$(PKG_VERSION)
+PKG_HASH:=613c4132cfed26ccd6e9f59a3e8590723f07969ac677f4957cbe000b13737fea
+
+PKG_MAINTAINER:=Ray Wang <r@hev.cc>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=License
+
+PKG_BUILD_FLAGS:=no-mips16
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/hev-socks5-tunnel
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=A high-performance tun2socks for Unix
+  URL:=https://github.com/heiher/hev-socks5-tunnel
+endef
+
```

Comment:
Description for this package would be nice to have.

Reply:
I have already added it. Thanks.